docs/_data/bearer_scan.yaml

```
@@ -160,6 +160,12 @@ options:
     usage: Specify which severities are included in the report.
     environment_variables:
       - BEARER_SEVERITY
+  - name: skip-git-ignore
+    default_value: "false"
+    usage: |
+      Do not automatically skip files that match patterns in .gitignore
```

Suggestion: it's a bit clearer to me than a "do not" that I must set to `true` to enable. Take it or leave it, just an alternative option. 

```suggestion
      Scan files even if their paths match patterns in .gitignore
```